config/presets/standalone.go

```diff
@@ -80,7 +80,9 @@ func standalone() config.Config {
 	conf.POET.GracePeriod = 12 * time.Second
 	conf.POET.CycleGap = 30 * time.Second
 	conf.POET.PhaseShift = 30 * time.Second
-	conf.POET.RequestTimeout = 12 * time.Second // RequestRetryDelay * 2 * MaxRequestRetries*(MaxRequestRetries+1)/2
+	conf.POET.PositioningATXSelectionTimeout = 25 * time.Minute
```

Comment:
Same here, the `GracePeriod` is 12 seconds, the timeout for ATX Selection must be smaller than that, I suggest `10 * time.Second`.